draft-ietf-rtcweb-jsep.xml

```diff
@@ -2146,12 +2146,17 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             lines are not meaningful in the JSEP context and
             MAY be discarded once they have been checked.
             <list>
+              <t>The "c=" line MUST be checked for syntax but its value SHOULD be ignored.
+              This supersedes the guidance in <xref target="RFC5245"/>, Section 6.1,
+              to use "ice-mismatch" to indicate mismatches between "c=" and
+              the candidate lines. Because JSEP always uses ICE--and generally
+              uses trickle ICE--"ice-mismatch" is not useful in this case.
```

Comment:
Give the state of trickle ice, I should like to remove the "--and generally uses trickle ICE--".